### ISSUE TITLE
Terminal - Root password

### ISSUE THREAD

**Issue by MTrinkiesJr** [pelya/commandergenius#58]
I want to use the terminal to install gnome. Obvously I need the Root password. Could you help me?

**Comment by pelya** [comment]
Use sudo instead of su

**Comment by MTrinkiesJr** [comment]
Thanks this works but is there no root password?

**Comment by pelya** [comment]
Yes, there is no password, I'll patch su in the next release so it won't
ask it.

**Comment by MTrinkiesJr** [comment]
Very Nice! This distribution is perfect. Thanks for your work!
